butterknife-compiler/src/main/java/butterknife/compiler/ButterKnifeProcessor.java

```diff
@@ -1348,6 +1354,21 @@ private static AnnotationMirror getMirror(Element element,
     return null;
   }
 
+  /**
+   * Perform two lookups to see if the androidx annotation and core libraries are on the classpath.
+   * If both aren't present butterknife will leverage support annotations and compat libraries
+   * instead.
+   */
+  static boolean hasAndroidX() {
+    try {
+      Class.forName("androidx.annotation.NonNull");
+      Class.forName("androidx.core.content.ContextCompat");
```

Comment:
These are unlikely to be on the same classpath as the annotation processor. We should look them up from `elements.getTypeElement` to use the application classpath.

Reply:
92c115c